packages/diagrams/frontend/sirius-components-diagrams/src/renderer/DiagramRenderer.tsx

```diff
@@ -204,7 +204,7 @@ export const DiagramRenderer = ({ diagramRefreshedEventPayload }: DiagramRendere
         });
       }
     },
-    [setNodes, targetNodeId, draggedNode?.id]
+    [setNodes, targetNodeId, draggedNode?.id, diagramRefreshedEventPayload.id]
```

Comment:
The function does not actually use `diagramRefreshedEventPayload.id`. What it *does* use is the `layoutOnBoundsChange` function which _itself_ depends on `diagramRefreshedEventPayload.id`. Shouldn't `layoutOnBoundsChange` itself be in the dependency array?

Reply:
Yes, you're right, it's clearer to depend directly on `layoutOnBoundsChange` rather than the `diagramRefreshedEventPayload.id`
I changed it